### PR TITLE
Some_blog : processing data in models

### PR DESCRIPTION
**Pull request by ambrose-kibet**
## In this milestone we:
- [ ] Add migration files to create tables 
- [ ] run migration files to create tables.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+ActiveRecord::Schema[7.0].define(version: 2023_05_18_084327) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.text "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "user_id"
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Kindly rename this table column to `author_id` in keeping with the ERD diagram shared with you.

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    t.bigint "post_id"
+    t.index ["post_id"], name: "index_comments_on_post_id"
+    t.index ["user_id"], name: "index_comments_on_user_id"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "user_id"
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Kindly rename this table column to `author_id` in keeping with the ERD diagram shared with you.

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    t.index ["user_id"], name: "index_likes_on_user_id"
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.string "title"
+    t.text "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "commentscounter"
+    t.integer "likescounter"
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Kindly rename these table columns to `comments_counter` and `likes_counter` as that is the convention for naming columns in Rails. Distinct words are separated with an underscore.

File: db/schema.rb [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    t.index ["author_id"], name: "index_posts_on_author_id"
+  end
+
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.text "photo"
+    t.text "bio"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "postsCounter"
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Kindly rename this column to `posts_counter` based on naming conventions.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @ambrose-kibet,

Good job so far!
✔️ Use of PostgreSQL for database
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/schema.rb (hunk 1 above)
Comment (ambrose-kibet) [reply db/schema.rb]: @TedLivist  it is named according to erd diagran
erdcomments

File: db/schema.rb (hunk 2 above)
Comment (ambrose-kibet) [reply db/schema.rb]: @TedLivist  again named according to erd diagram
sameerd

Comment (Davidosky007) [reply db/schema.rb]: * Kindly follow the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) provided in the project requirement, You will see the previous reviewer is right, 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @ambrose-kibet 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Database migration files used to maintain database schema 👍 
* Good looking readme 👍 
* Well written PR 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidosky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi  @ambrose-kibet,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions
- [ ] Kindly remove the `Live demo` section from your README file altogether if the project is not yet deployed

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**